tasks/fix-cat2.yml

```diff
@@ -1166,15 +1168,18 @@
       - notimplemented
 
 - name: "MEDIUM | RHEL-07-030330 | PATCH | The operating system must immediately notify the System Administrator (SA) and Information System Security Officer ISSO (at a minimum) when allocated audit record storage volume reaches 75% of the repository maximum audit record storage capacity."
-  command: "true"
-  changed_when: no
+  lineinfile:
+      path: /etc/audit/auditd.conf
+      regexp: ^space_left\s*=\s*
+      line: space_left = {{ ( ansible_mounts | json_query(query) / 1024 / 1024 ) | int + 1 }}
```

Comment:
What is the outcome of this? It looks like it converts the volume size to MB then converts that to an `integer` type and adds 1? So if the volume is 500MB it would set the space left to 501MB? or am I not reading this right. 

If we take this route of automatically gathering the /var/log/audit volume size we should do something like `{{ ( ansible_mounts | json_query(query) * 0.25 / 1024 / 1024 ) | int }}` to get 25% of the total size?

Note: not sure if the above filter syntax actually works its just an example.

I also like the method @mikerenfro uses in #85 allowing a user to set a static value in vars file. Maybe we can move this entire filter line to the vars file making it easier for a user to override if they want? i.e.

in `defaults/main.yml` set up a var like in #85. 
```
# RHEL-07-030330: Use the automatic space left calculation based on /var/log/audit volume size or set this to a custom value that is at least 25% of the free space in /var/log/audit (measured in megabytes)
rhel7stig_auditd_space_left: {{ ( ansible_mounts | json_query(query) * 0.25 / 1024 / 1024 ) | int }}
```

Then in this task:
```
line: space_left = {{ rhel7stig_auditd_space_left }}

Reply:
Good catch, it does look like I didn't use 25%, oops. I'm fine moving it to the defaults file.

Reply:
If you want to merge #85, I can rebase this PR.

Reply:
Ok I merged #85